test/vcr_cassettes/unit_test_github_basic_detective.yml

```diff
@@ -148,6 +148,85 @@ http_interactions:
s/assimilation/assimilation-official/commits/07d6b155c90bd096c549e7e2b08451f72c1393a1"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMQ=="},{"name":"v0.1.0","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/v0.1.0","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/v0.1.0","commit":{"sha":"1cf88e2488e716b8af74e4557212f50ae9181f7e","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/1cf88e2488e716b8af74e4557212f50ae9181f7e"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMA=="},{"name":"v0.1.0-RC4","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/v0.1.0-RC4","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/v0.1.0-RC4","commit":{"sha":"8d107cb298a184755886d271b7152c48cf8c9fc6","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/8d107cb298a184755886d271b7152c48cf8c9fc6"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMC1SQzQ="},{"name":"v0.1.0-RC3","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/v0.1.0-RC3","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/v0.1.0-RC3","commit":{"sha":"db74a1bfb09f8dc11f68b51f2fbc3ed97a5612bd","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/db74a1bfb09f8dc11f68b51f2fbc3ed97a5612bd"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMC1SQzM="},{"name":"v0.1.0-RC2","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/v0.1.0-RC2","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/v0.1.0-RC2","commit":{"sha":"bb357a0ec1eba229f55a38973323dd5274e63dac","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/bb357a0ec1eba229f55a38973323dd5274e63dac"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMC1SQzI="},{"name":"v0.1.0-RC1","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/v0.1.0-RC1","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/v0.1.0-RC1","commit":{"sha":"e15cc4ad7570f4db66157fa24dd8c38dba80c474","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/e15cc4ad7570f4db66157fa24dd8c38dba80c474"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzL3YwLjEuMC1SQzE="},{"name":"1.0.1","zipball_url":"https://api.github.com/repos/assimilation/assimilation-official/zipball/refs/tags/1.0.1","tarball_url":"https://api.github.com/repos/assimilation/assimilation-official/tarball/refs/tags/1.0.1","commit":{"sha":"0bb087786c96a10935dbc32055c5c76cde9bc6b2","url":"https://api.github.com/repos/assimilation/assimilation-official/commits/0bb087786c96a10935dbc32055c5c76cde9bc6b2"},"node_id":"MDM6UmVmNDIzNzMwNDY6cmVmcy90YWdzLzEuMC4x"}]'
```

Comment:
This is cheating a bit: manually added some tags to have a more realistic unit test. if someone could add a version tag to https://github.com/ciitest/test-repo then we can use the 'real' response here.